meta-xt-domx/recipes-extended/xen/xen_git.bbappend

```diff
@@ -1,5 +1,7 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
 require xen-source.inc
 
-do_configure:append() {
-    oe_runmake xt_defconfig
```

Comment:
Could you please describe why are you doing this change?

Reply:
I described this in commit message